### PR TITLE
feat: buzzsaw, rumble, and ignition improvements

### PR DESCRIPTION
**Pull request by hotshotp**
Fixes #[issue_no]

## Summary of Changes

Improved engine interior buzzsaw to be a little more crisp and defined, and also same with the rumble sound.

Also improved the interior engine ignition sound:
There are now 2 random ignition samples, and two of the same ones will never play more than once in a row

Added cone attenuation to 50% throttle whine

This also adds PTU in cockpit and volume slider code which will not take place until the EFB settings page gets merged
## Additional context

Discord username (if different from GitHub):

Boris#9134

## Testing instructions

1. Start engines and go to cabin view. Listen to fuel flow intro/ignition.
2. Takeoff and listen to the new buzzsaw and rumble, make sure there are no artifacts, weird chorusing, or pops
3. When landing, pay attention to the sound when lowering the nose to the ground
4. On exterior view, go to 50% thrust and go in front and behind the aircraft, see that the whine is quieter in the back

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -1232,36 +1245,6 @@
...
-
-        <Sound WwiseData="true" WwiseEvent="ptucockpit" NodeName="PEDALS_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
-            <Requires SimVar="TURB ENG N1" Units="PERCENT" Index="2">
-                <Range UpperBound="1" />
-            </Requires>
-            <Requires LocalVar="A32NX_SOUND_PTU_AUDIBLE_COCKPIT" Units="BOOL">
-                <Range LowerBound="1" />
-            </Requires>
-        </Sound>
-
```
Comment (pepperoni505) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:1264]: Why did you remove these? Weren't these just added?

Comment (hotshotp) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:1264]: they caused problems with the current ptu sound for some odd reason, so I removed and remade wwise events with a different name

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 2]
```diff
@@ -975,6 +975,19 @@
...
+		<Requires SimVar="TURB ENG N1" Units="PERCENT" Index="1">
+			<Range UpperBound="1" />
+		</Requires>
+	    </Sound>
+	
+	    <Sound WwiseData="true" WwiseEvent="ptu_sound" NodeName="SOUND_ENGINE_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
+		<Requires SimVar="TURB ENG N1" Units="PERCENT" Index="2">
+			<Range UpperBound="1" />
+		</Requires>
+	    </Sound>
```
Comment (pepperoni505) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:989]: Shouldn't you use both left and right? Or maybe a node in the center?

**Review by pepperoni505 — APPROVED** [review]
LGTM

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml (hunk 2 above)
Comment (hotshotp) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:989]: they are not orientation positional, the speaker panning stays the same

**Comment by NonstopFlying** [comment]
Category: QA Trial
Name: NonstopFlying #6771
Date of testing: Current date 17/02/2021
Version of the sim: 1.13.16.0
PR Tested: #3548
Tier of testing: 2
Changes to observe:
Buzzsaw form the interior 
Ignition samples 
50% whine from the back and front
front wheel on landing

Testing technique: Landed and started the airplane multiple times in Newcastle 
Steps you've made to test the Pull Request:
1. Spawned in at Newcastle and started the engines 4 times from the cabin
2. Tookoff 3 times 
3. Landed 3 times
4. Listened to the different sounds

Overall Rating: Good
Conclusions of the testing: Starting sound didn't repeat after each other meaning the different samples always played on a 1,2,1,2,1 sequence,
the landing sound is amazing I love it, as well as the 50% whine which I can clearly tell is quiter from the back.

**Comment by sidnov** [comment]
Category: QA Tester
Name: sidnov#8337
Date of testing: 17.2.2021
Version of the sim: 1.13.16.0
PR Tested: #3548 
Tier of testing: 2

### Changes to observe: Sound improvements

### Testing technique: Tryout

### Steps you've made to test the Pull Request:

1. Spawned at gates, C/D. Started engine 2 in the cabin view
2. Observed the improved sounds on startup/ fuel flow
3. Spawned on the runway, set engine thrust 50%. Observed a quieter whining sound from the back of the aircraft
4. Started rolling on the runway, observed new rumble sound.
5. On rotation, observed a clearer buzzsaw sound.
6. Spawned on an approach for RWY27, observed nose touchdown sound.

### Overall Rating: Good
### Conclusions of the testing: More new sounds and improvements, a great one again by Boris! :)

**Comment by Benjozork** [comment]
Please document new simvars in `simvars.md`
